src/vivarium_csu_alzheimers/constants/data_keys.py

```diff
@@ -31,19 +32,22 @@ def log_name(self):
 
 
 class __Alzheimers(NamedTuple):
-    # Prevalence scale factor will scale the population and "fertility" of the model
-    PREVALENCE_SCALE_FACTOR: str = (
-        "cause.alzheimers_disease_and_other_dementias.prevalence_scale_factor"
-    )
-    # Prevalence will just be set to 1 so all simulants are created with the disease
     PREVALENCE: str = "cause.alzheimers_disease_and_other_dementias.prevalence"
+    BBBM_CONDITIONAL_PREVALANCE: str = "cause.bbbm.prevalence"
+    MCI_CONDITIONAL_PREVALENCE: str = "cause.mci.prevalence"
     INCIDENCE_RATE: str = "cause.alzheimers_disease_and_other_dementias.incidence_rate"
+    STATE_INCIDENCE_RATE: str = (
```

Comment:
I think this name is confusing, and hard to match to the `i_AD` name in the docs